src/main/java/net/mcreator/ui/dialogs/NewVariableDialog.java

```diff
@@ -28,11 +28,12 @@
 
 import javax.swing.*;
 import java.awt.*;
+import java.util.Objects;
 
 public class NewVariableDialog {
 
 	public static VariableElement showNewVariableDialog(JFrame frame, boolean showScope,
-			OptionPaneValidatior variableNameValidator, VariableElementType... supportedTypes) {
+			OptionPaneValidatior variableNameValidator, String... supportedTypes) {

```

Comment:
You changed method signature but callers of this method were not updated as far as I can see

Reply:
![image](https://user-images.githubusercontent.com/38427877/116112625-b99f0600-a685-11eb-9406-5412896b2269.png)
No, they have been updated.